src/renderer/src/pages/game-details/hero-panel.tsx

```diff
@@ -47,27 +47,29 @@ export function HeroPanel({
   } = useDownload();
   const isGameDownloading = isDownloading && gameDownloading?.id === game?.id;
 
-  const updateLastTimePlayed = useCallback(() => {
-    setLastTimePlayed(
-      formatDistance(game.lastTimePlayed, new Date(), {
-        addSuffix: true,
-      })
-    );
-  }, [game?.lastTimePlayed, formatDistance]);
-
   useEffect(() => {
     if (game?.lastTimePlayed) {
-      updateLastTimePlayed();
+      setLastTimePlayed(
+        formatDistance(game.lastTimePlayed, new Date(), {
+          addSuffix: true,
+        })
+      );
+    }
+  }, [game?.lastTimePlayed, formatDistance]);
 
-      const interval = setInterval(() => {
-        updateLastTimePlayed();
-      }, 1000);
+  const formatPlayTime = (milliseconds: number) => {
+      const seconds = milliseconds / 1000
+      const minutes = seconds / 60
 
-      return () => {
-        clearInterval(interval);
-      };
-    }
-  }, [game?.lastTimePlayed, updateLastTimePlayed]);
+      if (minutes < 120) {
+        return minutes.toFixed(0) + " " + t("minutes")
+      }
+
+      const numberFormat = new Intl.NumberFormat(i18n.language, { maximumFractionDigits: 1 })
```

Comment:
You should create the `NumberFormat` instance outside of the component so it doesn't have to instantiated every single time

Reply:
Not sure what's the best way to solve this as I don't know much of React, but I used a `useMemo` hook and put the `NumberFormat` inside it, what do you think?